docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -14,19 +14,69 @@ helpviewer_keywords:
 
 This article shows how to migrate from [Newtonsoft.Json](https://www.newtonsoft.com/json) to <xref:System.Text.Json>.
 
- `System.Text.Json` focuses primarily on performance, security, and standards compliance. It has some key differences in default behavior and doesn't aim to have feature parity with `Newtonsoft.Json`. For some scenarios, `System.Text.Json` has no built-in functionality, but there are recommended workarounds. For other scenarios, workarounds are impractical. If your application depends on a missing feature, consider [filing an issue](https://github.com/dotnet/runtime/issues/new) to find out if support for your scenario can be added.
+`System.Text.Json` focuses primarily on performance, security, and standards compliance. It has some key differences in default behavior and doesn't aim to have feature parity with `Newtonsoft.Json`. For some scenarios, `System.Text.Json` has no built-in functionality, but there are recommended workarounds. For other scenarios, workarounds are impractical. If your application depends on a missing feature, consider [filing an issue](https://github.com/dotnet/runtime/issues/new) to find out if support for your scenario can be added.
 
 <!-- For information about which features might be added in future releases, see the [Roadmap](https://github.com/dotnet/runtime/tree/81bf79fd9aa75305e55abe2f7e9ef3f60624a3a1/src/libraries/System.Text.Json/roadmap/README.md). [Restore this when the roadmap is updated.]-->
 
 Most of this article is about how to use the <xref:System.Text.Json.JsonSerializer> API, but it also includes guidance on how to use the <xref:System.Text.Json.JsonDocument> (which represents the Document Object Model or DOM), <xref:System.Text.Json.Utf8JsonReader>, and <xref:System.Text.Json.Utf8JsonWriter> types. The article is organized into sections in the following order:
 
+* [Table of differences between Newtonsoft.Json and System.Text.Json](#table-of-differences-between-newtonsoftjson-and-systemtextjson)
```

Comment:
This list pretty much just duplicates the automatic in-article TOC.

Reply:
You don't get an in-article TOC for h3 headings.

Reply:
But these are H2s right?

![image](https://user-images.githubusercontent.com/24882762/72560031-0416bd80-385b-11ea-9609-55d28f73ab7a.png)

Reply:
Right, sorry, I thought you were talking about the table. I'll reconsider this, it made sense earlier when I had all h2s and it explained the groupings. Now it isn't as useful as it was.

Reply:
That was a good catch, thanks -- I did decide to remove the extra toc.